imports/plugins/core/ui-navbar/client/components/navbar/navbar.html

```diff
@@ -6,8 +6,9 @@
     <div class="menu">
       {{> tagNav tagNavProps}}
     </div>
-    
-    <div class="search">{{> React IconButtonComponent}}</div>
+    {{#if isSearchEnabled}}
```

Comment:
Because I already needed to query on the db to get the template and I thought it made the code there easier to read.